gprofiler/metadata/application_identifiers.py

```diff
@@ -273,6 +273,21 @@ def get_app_id(self, process: Process) -> Optional[str]:
         return None
 
 
+class _NodeModuleApplicationIdentifier(_ApplicationIdentifier):
+    def get_app_id(self, process: Process) -> Optional[str]:
+        skip = False
+        for arg in process.cmdline()[1:]:
+            if skip:
+                skip = False
+                continue
+            if arg in ["--require", "-r"]:
+                skip = True
+                continue
+            if arg.endswith(".js"):
+                return f"nodejs: {arg}"
```

Comment:
Let's use `_append_file_to_proc_wd` here like used in others, e.g `_PythonModuleApplicationIdentifier`. The idea is that the final appid will be `nodejs: {file_arg} ({abs path of file arg})`.

Reply:
(See tests use of `process_with_cmdline` and `PROCESS_CWD`)